lib/services/alexa/request/lifecycle/initialize.ts

```diff
@@ -23,6 +23,11 @@ const utilsObj = {
 export const initializeGenerator = (utils: typeof utilsObj) => async (runtime: AlexaRuntime, input: AlexaHandlerInput): Promise<void> => {
   const { requestEnvelope } = input;
 
+  // Identify on analytics system
+  if (runtime?.services?.AnalyticsClient) {
+    runtime.services.analyticsClient.identify(runtime.getVersionID());
+  }
+
```

Comment:
```suggestion
    runtime.services.analyticsClient.identify(runtime.getVersionID());
```
as it's not null anymore

Reply:
done